common/src/main/java/org/dash/wallet/common/util/GenericUtils.java

```diff
@@ -186,12 +194,20 @@ public static String formatFiatWithoutComma(String fiatValue){
 
     public static String fiatToString(Fiat fiat) {
         MonetaryFormat format = Constants.INSTANCE.getSEND_PAYMENT_LOCAL_FORMAT().noCode();
+        return fiatToString(fiat, format);
+    }
+
+    public static String fiatToStringRoundUp(Fiat fiat) {
+        MonetaryFormat format = Constants.INSTANCE.getSEND_PAYMENT_LOCAL_FORMAT().noCode().roundingMode(RoundingMode.UP);
+        return fiatToString(fiat, format);
+    }
```

Comment:
Currently, this is used for discounted values only and the purpose is to round up to ensure that the calculations agree.  In both cases below, the discount was 2.75% when the user entered $50.  The enter amount screen was showing the discounted value of 48.62 while the payment confirmation dialog shows 48.63.

Displayed |  | internal (long) | Description
-- | -- | -- | --
48.62 |   | 4862499917 | calculated from DASH value and converted to fiat using an exchange rate
48.63 |   | 4862500000 | calculated from an exact fiat value of 50.00

If we round up, then both will match, which means that the user will see 48.63 both times in the flow.